playwright/test/concept.test.ts

```diff
@@ -151,3 +156,106 @@ test.describe('concepts', () => {
     );
   });
 });
+
+const allRecordsLink = async (
```

Comment:
I added these two helpers to improve the storytelling ability of the tests that use them - (e.g. with `showsTab`, it is clear that the test is looking for the two tabs, rather than one button element with some content and another button element with some other content).

I have left the existing tests in this file alone, but if desired, I will happily extend the same style to them.

I'm aware that (particularly with playwright-style tests) this kind of concealment of the detail is not to everyones taste.